internal/server/index.go

```diff
@@ -16,7 +16,7 @@ var alphaNumericRunes = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUV
 // ServeIndex serve the index.html file with nonce generated for supporting
 // restrictive CSP while using material-ui from the embedded virtual filesystem.
 //go:generate broccoli -src ../../public_html -o public_html
-func ServeIndex(publicDir, base, rememberMe, resetPassword string) fasthttp.RequestHandler {
+func ServeIndex(publicDir, base, theme, rememberMe, resetPassword string) fasthttp.RequestHandler {
```

Comment:
```suggestion
func ServeIndex(publicDir, base, rememberMe, resetPassword, theme string) fasthttp.RequestHandler {
```

This is a nit more than anything but can we alphabetise these?